lib/screens/config/v2/dup.ex

```diff
@@ -21,7 +21,22 @@ defmodule Screens.Config.V2.Dup do
     children: [
       primary_departures: Departures,
       secondary_departures: Departures,
-      header: CurrentStopId,
       evergreen_content: {:list, EvergreenContentItem}
     ]
+
+  defp value_from_json("header", %{"stop_id" => _} = header) do
+    CurrentStopId.from_json(header)
+  end
+
+  defp value_from_json("header", %{"stop_name" => _} = header) do
+    CurrentStopName.from_json(header)
+  end
```

Comment:
Just sanity checking myself here: would this mean that we prefer `stop_id` over `stop_name` based on the ordering of the pattern matching, or are we assuming that there will always only be either a `stop_id` _or_ `stop_name`?